html/apps/tweetfeed/css/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  background-color: gray !important;
+}
+
+md-content.md-default-theme, md-content {
+    color: rgba(0,0,0,0.87);
+    background-color: rgb(66, 66, 66);
+}
+
+img.md-user-avatar{
+  border-radius: 25%;
+}
+
+md-input-container{
+  margin-top: 3%;
+}
+
+#content-scroller{
+  overflow: visible;
+}
+
+md-toolbar button.md-button.md-icon-button:first-child {
+  margin-left: 0px;
+  margin-right: 16px;
+}
+
+md-icon{
+  height: 20px !important;
+  margin: 0 !important;
+}
+md-icon svg {
+    pointer-events: none;
+    display: block;
+    color: rgba(0,0,0,0.26);
+}
+
+.card-title {
+  font-size: 20px !important;
+  letter-spacing: 0.005em;
+  display: block;  
+  font-weight: bold;
+  margin-left: 20px;
+  margin-top: 10px;
+}
+
+.card-subhead {
+  font-size: 14px;
+  letter-spacing: 0.010em;
+  line-height: 24px;
+  color: gray;
+  margin-left: 20px;
+}
+
+.card .inside .description.favs {
+    color: rgba(85, 85, 85, 0.5);
+    margin-top: 20px;
+}
+  
+md-toolbar button.md-button.md-icon-button:first-child md-icon {
+  color: white;
+}
+
+
+.fit {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+#summary {
+  margin: 32px 72px;
+}
+.card {
+  background: rgb(224,224,224);
+  min-width: 100px;
+  min-height: 100px;
+  max-width: 400px;
+  margin: 8px;
+  -webkit-transition: .4s;
+  transition: .4s;
+  opacity: 0;
+  -webkit-transform: scale(0.75) translate3d(0, 200px, 0);
+  transform: scale(0.75) translate3d(0, 200px, 0);
+  border-radius: 2px;
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.14);
+  overflow: hidden;
+  -webkit-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+.card.angular-grid-item {
+  opacity: 1;
+  -webkit-transform: scale(1) translate3d(0, 0, 0);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The property -ms-transform is compatible with -webkit-transform and should be included as well.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2597868539/issues/source?bid=3288751&fileBranchId=3307409#l95)